.github/workflows/conformance.yml

```diff
@@ -63,7 +64,7 @@ jobs:
         uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81 # v5.5.1
         with:
           images: |
-            name=ghcr.io/nginxinc/nginx-gateway-fabric/nginx
+            name=ghcr.io/nginxinc/nginx-gateway-fabric/${{ matrix.nginx-image }}
```

Comment:
Will this work for plus since we don't push the plus image to ghcr?

Reply:
Yep, because we're not pulling the images, we're loading them, so it's just a local tag